AmplifyPlugins/API/AWSAPICategoryPlugin/AWSAPIPlugin.swift

```diff
@@ -44,7 +44,6 @@ final public class AWSAPIPlugin: NSObject, APICategoryPlugin, AWSAPIAuthInformat
 
```

Comment:
the comment above for `iReachabilityMap` no longer makes sense
```
/// The underlying map that retains instances of NetworkReachabilityNotifier.  Using a computed property
    /// to work around @available for use on stored properties
    var iReachabilityMap: [String: Any]?
```

if we no longer need `@available`, then we also don't need to use `iReachabilityMap`. can you capture this work, as a follow up if needed since this PR does have a lot of files change?

Reply:
I will create a new PR to remove `iReachabilityMap` and link it here.